js/src/Builder.js

```diff
@@ -495,6 +526,17 @@ function _set_mode (mode) {
   if (mode == 'rotate')
     this.map.deselect_text_labels()
   this.map.draw_everything()
+
+  if(mode === 'difference'){ // TODO: is not a mode button any more...
```

Comment:
Rather than set this up as a 'mode', we should make it something you choose from the menu, like the search bar. No need to put a button in the sidebar.